lib/checks/canFindDenominatorInNumerator.js

```diff
@@ -20,41 +33,70 @@ function canFindDenominatorInNumerator(node) {
   if (Node.Type.isParenthesis(denominator)) {
     denominator = denominator.content;
   }
-  if (!(numerator.op === '+' || numerator.op === '-' ||
-        denominator.op === '+' || numerator.op === '-')) {
+
+  let numeratorArgsLength;
+  // If numerator is '*' op, it signifies a single 'ax', should be assigned a
+  // length of 2
```

Comment:
I think a clearer way to do this check is to see if it's a polynomial term (and then length 1) or else make it `numerator.args.length`

also if numerator doesn't have args, you should return false, which you don't do now right?